src/main/scala/mesosphere/marathon/core/volume/Volumes.scala

```diff
@@ -0,0 +1,97 @@
+package mesosphere.marathon.core.volume
```

Comment:
I'll move implementation specifics to an impl package. There's lots of cleanup to do here